src/utils.jl

```diff
@@ -232,15 +243,28 @@ function fill_quadratic_constraint_caches!(
     if !isempty(quad_aff_vars)
         model.quadratic_constraint_cache_pv[new_ci] = quad_aff_vars
     end
+
     if !isempty(quad_params)
         model.quadratic_constraint_cache_pp[new_ci] = quad_params
     end
+
     if !isempty(aff_params)
         model.quadratic_constraint_cache_pc[new_ci] = aff_params
     end
+
     if !isempty(terms_with_variables_associated_to_parameters)
         model.quadratic_constraint_variables_associated_to_parameters_cache[new_ci] =
             terms_with_variables_associated_to_parameters
     end
     return nothing
 end
+
+function quadratic_constraint_cache_map_check(
+    model::ParametricOptimizer,
+    idxs::MOI.ConstraintIndex{F,S},
+) where {F,S}
+    cached_constraints = values(model.quadratic_added_cache)
+    # Using this becuase some custom brodcast method throws errors if
+    # inner_idex .∈ cached_constraints is used
+    return [i ∈ cached_constraints for i in idxs]
```

Comment:
Perhaps `in.(idxs, cached_constraints)`?

Reply:
It is the same thing. It is also quite slow since it checks all cached constraints vs all the idx's probably something smarter can be done.

Reply:
What is `for i in idxs` doing ? `idxs` is not an iterable, it's a constraint index.

Reply:
we want parameter * variable.
We can change things if needed.
POI takes note it there are parameter * variable, so if your code does not have it, you should see no degradation.

Reply:
@blegat idxs is an iterable of constraint indices.